Apps/Playground/Win32/App.cpp

```diff
@@ -107,18 +112,25 @@ namespace
         graphics = Babylon::Graphics::CreateGraphics<void*>(hWnd, width, height);
         graphics->StartRenderingCurrentFrame();
 
+        auto usingV8Inspector = false;
+#ifdef V8_ENGINE
+        usingV8Inspector = Babylon::AppRuntimeV8::EnableInspector(5642);
+#endif
+
         runtime = std::make_unique<Babylon::AppRuntime>();
         inputBuffer = std::make_unique<InputManager<Babylon::AppRuntime>::InputBuffer>(*runtime);
 
-        runtime->Dispatch([width, height, hWnd](Napi::Env env) {
+        runtime->Dispatch([width, height, hWnd, usingV8Inspector](Napi::Env env) {
             graphics->AddToJavaScript(env);
-          
-#ifndef V8_USE_INSPECTOR
-            // Initialize console plugin.
-            Babylon::Polyfills::Console::Initialize(env, [](const char* message, auto) {
-                OutputDebugStringA(message);
-            });
-#endif
+            
+            // If we've initialized the inspector server, console messages 
+            // will be sent to the DevTools console.
+            if (!usingV8Inspector)
+            {
+                Babylon::Polyfills::Console::Initialize(env, [](const char* message, auto) {
```

Comment:
Can this be detected inside the Console initialize? For example, in the WIndow polyfill, we check to see if the window already has the method before overriding it. Can we do the same here?

Reply:
Checking if `console` is defined feels like the better option here. One thing I'll call out is that V8 defines the console object no matter what, so even if we don't use the inspector we won't see messages printed to the console. Maybe I can add another optional parameter called `overrideEngine` that defaults to false, just in case you really want your console callback called?

Reply:
I think if there is already a console object, then we should shim it instead of replace it. The console.log calls the callback and continue to call the existing console.log function? JavaScriptCore also defines a console object. I wonder if this will make it work in all cases.

Reply:
Added commit https://github.com/BabylonJS/BabylonNative/pull/607/commits/8b6ebb6d7e1d3d1d1ac127dd9a0a74ac2d53c70d shimming the console functions